wgpu-core/src/device/mod.rs

```diff
@@ -534,6 +558,13 @@ impl<B: GfxBackend> Device<B> {
             .allocate(&self.raw, requirements, mem_usage)?;
         block.bind_buffer(&self.raw, &mut buffer)?;
 
+        let mut uninitialized_ranges = range_alloc::RangeAllocator::new(Range {
+            start: 0,
+            end: desc.size,
+        });
+        // Mark buffer as uninitialized
+        let _ = uninitialized_ranges.allocate_range(desc.size);
```

Comment:
would reversing this works? i.e. storing `initialized_ranges` instead of uninitialized

Reply:
see "RangeAllocator not a great fit" comment further down